app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,25 @@
+# show each individual question
+get '/questions/:id' do
+  question = Question.find(params[:id]).next
+  survey = TakenSurvey.find(params[:survey_id])
+  if defined?(question)
+    erb :'/questions/show', locals: {question: question, survey: survey}
+  else
+    erb :'/answers'
+  end
+end
+
+# post to the answers table to track a users response for a specific question then redirect to the next question, else return to the same page and
+post '/questions' do
+  answer = Answer.new(params)
+  if answer.save
+    # question id is static here so no way of adding a method to the model. It will need to be manually incremented (+1) here or in the view
+    redirect '/questions/<%= params[:question.id] %> + 1'
```

Comment:
On line 3 you are incrementing to the next question after incrementing to the next question here...so you will be

(question + 1).next